src/components/header/Backtrace.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import React, { PureComponent } from 'react';
```

Comment:
please use the new convention of `import * as React from 'react'` and use `React.PureComponent` below.

Reply:
Ok, sure.